src/Codeception/Command/Shared/FileSystemTrait.php

```diff
@@ -51,8 +51,8 @@ protected function completeSuffix(string $filename, string $suffix): string
 
     protected function removeSuffix(string $classname, string $suffix): string
     {
-        $classname = preg_replace('~\.php$~', '', $classname);
-        return preg_replace("~$suffix$~", '', $classname);
+        $classname = preg_replace('#\.php$#', '', $classname);
+        return preg_replace("~{$suffix}$~", '', $classname);
```

Comment:
Is there a good reason to standardize on `#`? `/` is much more common.

Also you missed different delimiters in a few places - this one is the most glaring.

Reply:
> / is much more common.

Yes, however it is also too common to see slashes within the same expression but for another purpose.

> Is there a good reason to standardize on #?

The idea is to use a character that is easy to identify to increase readability.
It's a matter of choosing, but `/` would be my last pick.

> Also you missed different delimiters in a few places - this one is the most glaring.

Yes, sorry about that.
I try to do my best with these PRs, but there are a few things I still tend to skip. That is the goal of this PR itself and that is why I value reviews so much.

Reply:
@Naktibalda I just did a commit to replace the ones that I may have missed

Reply:
The line above this comment hasn't been updated.